opensrp-app/src/main/java/org/smartregister/executor/CoreLibraryExecutors.java

```diff
@@ -0,0 +1,77 @@
+package org.smartregister.executor;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.NonNull;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+/**
+ * <p>
+ * Global executor pools for plug-and-play CoreLibrary operations.
+ * <p>
+ * Grouping tasks like this avoids the effects of task starvation (e.g. disk reads don't wait behind
+ * webservice requests).
+ *
+ * Created by Ephraim Kigamba - nek.eam@gmail.com on 09-04-2020.
+ */
+public class CoreLibraryExecutors {
```

Comment:
There is already existing class `org.smartregister.util.AppExecutors`

Reply:
Fixed